### PR TITLE
Show similar modes to standard modes, clean up tests and some styles

### PR DESCRIPTION
**Pull request by ishythefishy**

### CODE REVIEW

File: apps/fretonator-web/src/app/util/constants.ts [hunk 1]
```diff
@@ -626,3 +626,20 @@ export const StringFrequencies = {
   'A': 110.00,
   'E': 82.41
 }
+
+export const StandardModePatterns = [
+  Mode.ionian,
```
[the remark this answers is not in the document]
Comment (ishythefishy) [reply apps/fretonator-web/src/app/util/constants.ts:631]: I’ve just had an idea for a better solution.

[the remark this answers is not in the document]
Comment (ishythefishy) [reply apps/fretonator-web/src/app/util/constants.ts:631]: Nah that was worse!

File: apps/fretonator-web/src/app/common/fret-map/fret-map.service.ts [hunk 2]
```diff
@@ -408,6 +411,40 @@ export class FretMapService {
...
+
+  getSimilarModes = (modeMap: ModeMap, inputMode: Mode): SimilarModes => {
+    const firstModeInPattern = StandardModePatterns.indexOf(inputMode);
+
+    if (firstModeInPattern === -1) {
+      return [];
+    }
 
+    const similarModes = modeMap
+      .map((noteObject, index) => {
```
Comment (geometricpanda) [inline apps/fretonator-web/src/app/common/fret-map/fret-map.service.ts:438]: Minor thing, you can use an es6 return syntax as this is a fat arrow function

File: apps/fretonator-web/src/app/util/constants.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (geometricpanda) [reply apps/fretonator-web/src/app/util/constants.ts:631]: Let’s get on a google hangout at some point this evening and we can go though this properly

**Pull request merged**